scheduler/src/cook/mesos/api.clj

```diff
@@ -148,22 +148,20 @@
    (s/optional-key :extract) s/Bool
    (s/optional-key :cache) s/Bool})
 
-(def Instance
-  "Schema for a description of a single job instance."
-  {:status s/Str
-   :task_id s/Uuid
-   :executor_id s/Uuid
-   :slave_id s/Str
-   :hostname s/Str
-   :preempted s/Bool
-   :backfilled s/Bool
-   :ports [s/Int]
-   (s/optional-key :start_time) s/Int
-   (s/optional-key :end_time) s/Int
-   (s/optional-key :reason_code) s/Int
-   (s/optional-key :output_url) s/Str
-   (s/optional-key :cancelled) s/Bool
-   (s/optional-key :reason_string) s/Str})
+(def Command
+  "Schema for a command"
+  {:value s/Str
+   (s/optional-key :async?) s/Bool
+   (s/optional-key :guard?) s/Bool
+   (s/optional-key :default?) s/Bool})
+
+(def CommandRequest
+  "Schema for a command in request"
+  {:value s/Str
+   (s/optional-key :name) s/Str
```

Comment:
My confusion is that it seems a user could add a "name" to their command map but it wouldn't be used.

This is fine though because I think a worthwhile enhancement will be to allow naming the commands and have the stderr and stdout use the name instead of the order so it would be like stdout.upload_to_elk which is a lot more interpret-able than stdout.1